rpm/0004-Introduce-mount-sd-service-that-is-executed-as-user.patch

```diff
@@ -48,19 +48,20 @@ index 758644be..9bce130a 100644
 -	@sed -e "s|\@udisksdprivdir\@|$(libexecdir)/udisks2|" $< > $@
 +	@sed -e "s|\@udisksdprivdir\@|$(libexecdir)/udisks2|" \
 +             -e "s|\@bindir\@|$(bindir)|" \
-+	     -e "s|\@localstatedir\@|$(localstatedir)|" \
+++	     -e "s|\@sharedstatedir\@|$(sharedstatedir)|" \
```

Comment:
There is an extra `+` sign, please remove it.

Reply:
Thanks, fixed.